packages/cli/src/lib/quire/index.js

```diff
@@ -127,28 +141,60 @@ async function initStarter (starter, projectPath) {
  * @param  {String}  version  Quire-11ty semantic version
  * @return  {Promise}
  */
-async function install(version='latest', options={}) {
+async function install(version, options={}) {
   console.debug(`[CLI:quire] installing quire-11ty@${version}`)
-  fs.ensureDirSync(INSTALL_PATH)
+  const absoluteInstallPath = path.join(__dirname, 'versions')
+  fs.ensureDirSync(absoluteInstallPath)
   /**
    * `Destination` is relative to `node_modules` of the working-directory
    * so we have included a relative path to parent directory in order to
    * install versions to a different local path.
    * @see https://github.com/scott-lin/install-npm-version
    */
-  const options = {
-    Destination: path.join('../', INSTALL_PATH, version),
+  const installOptions = {
+    Destination: path.join('../', version),
+    Debug: false,
     Overwrite: options.force || options.overwrite || false,
     Verbosity: options.debug ? 'Debug' : 'Silent',
+    WorkingDirectory: absoluteInstallPath
   }
-  await inv.Install(`${PACKAGE_NAME}@${version}`, options)
+  await inv.Install(`${PACKAGE_NAME}@${version}`, installOptions)
+
+  // delete empty `node_modules` directory that `install-npm-version` creates
+  const invNodeModulesDir = path.join(absoluteInstallPath, 'node_modules')
+  if (fs.existsSync(invNodeModulesDir)) fs.rmdir(invNodeModulesDir)
+
   symlinkLatest()
+
+  console.debug('[CLI:quire] installing dev dependencies')
+  /**
+   * Manually install necessary dev dependencies to run 11ty;
+   * these must be `devDependencies` so that they are not bundled into
+   * the final `_site` package when running `quire build`
+   */
+  const currentWorkingDirectory = cwd()
+  const versionDir = path.join(absoluteInstallPath, version)
+  chdir(versionDir)
+  await execaCommand(
+    'npm cache clean --force'
+  ).stdout.pipe(process.stdout)
+  await execaCommand(
+    'npm install --save-dev'
+  ).stdout.pipe(process.stdout)
```

Comment:
I do not think the standard out from these method calls will be displayed, we might remove it for now and revisit if we implement verbose output.
```suggestion
  await execaCommand('npm cache clean --force')
  await execaCommand('npm install --save-dev')
```